compose/ui/ui/src/webTest/kotlin/androidx/compose/ui/input/TextInputTests.kt

```diff
@@ -78,19 +73,28 @@ class TextInputTests : OnCanvasTests {
         }
     }
 
+    private fun WebApplicationScope.assertTextFieldValue(expected: String, actual: MutableState<TextFieldValue>, message: String? = null) {
```

Comment:
Maybe call it `assertTextEquals`?
- Not "TextFieldValue", because you're only looking at the text.
- "equals" because that's the convention.